arch/risc-v/src/mpfs/mpfs_fpga_canfd.c

```diff
@@ -425,7 +425,7 @@ static uint8_t g_rx_pool1[(sizeof(struct canfd_frame) + TIMESTAMP_SIZE) *
 
 /* (from interrupt) RX related functions */
 
-static void mpfs_can_retrieve_rx_frame(struct mpfs_driver_s *priv,
+static bool mpfs_can_retrieve_rx_frame(struct mpfs_driver_s *priv,
```

Comment:
I think I will change the bools to ints (OK / ERROR) for consitency with other functions in the same file